nevergrad/optimization/utils.py

```diff
@@ -147,7 +147,7 @@ def submit(self, fn: tp.Callable[..., tp.Any], *args: tp.Any, **kwargs: tp.Any)
 def _tobytes(x: ArrayLike) -> bytes:
     x = np.array(x, copy=False)  # for compatibility
     assert x.ndim == 1, f"Input shape: {x.shape}"
-    assert x.dtype == np.float
+    assert x.dtype == np.float, f"Incorrect type {x.dtype} is not float"
```

Comment:
can't not hurt to tell people what is going on when it's going wrong.

Reply:
definitely agree ;)